getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Installing Community
+published: true
+sorting: 80
+---
+
+These instructions describe how to download and install the latest version of CFEngine Community in a docker containerized environment using pre-compiled rpm packages for ubi9 images.
+
+It also provides instructions for the following:
+
+* **Install CFEngine on a policy server (hub) and on a Host (client).**
+A Policy Server (hub) is a CFEngine instance that contains promises (business policy) that get deployed to Hosts.
+Hosts are clients that retrieve and execute promises.
```

Comment:
Humm, not sure that we need to re-introduce the concept of a policy server / hub and Hosts.

```suggestion
```

Separately, maybe the glossary could benefit from attention but at least all of these come to mind.

- https://docs.cfengine.com/docs/3.24/overview-glossary.html#hub
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#host
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#client
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#cfengine-role
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#policy
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#promise
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#server
- https://docs.cfengine.com/docs/3.24/overview-glossary.html#policy-server